### PR TITLE
update create link function

### PR DESCRIPTION
**Pull request by nampx-0860**

### CODE REVIEW

**Comment by nampx-0860** [comment]
image

File: app/Repositories/Link/LinkRepository.php [hunk 1]
```diff
@@ -17,16 +17,16 @@ public function __construct(Link $link)
 
     public function createLinkUser()
     {
-        $strUser = md5(time().'user');
-        $tokenUser = substr($strUser, 0, 16);
+        $permitted_chars = '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ';
```
Comment (framgiaciv3) [inline app/Repositories/Link/LinkRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/Link/LinkRepository.php</br>**Line**: 20</br>Variable "permitted_chars" is not in valid camel caps format

File: app/Repositories/Link/LinkRepository.php [hunk 2]
```diff
@@ -17,16 +17,16 @@ public function __construct(Link $link)
...
+        $tokenUser = substr(str_shuffle($permitted_chars), 0, 16);
```
Comment (framgiaciv3) [inline app/Repositories/Link/LinkRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/Link/LinkRepository.php</br>**Line**: 21</br>Variable "permitted_chars" is not in valid camel caps format

File: app/Repositories/Link/LinkRepository.php [hunk 3]
```diff
@@ -17,16 +17,16 @@ public function __construct(Link $link)
...
 
         return $tokenUser;
     }
 
     public function createLinkAdmin()
     {
-        $strAdmin = md5(time().'admin');
-        $tokenAdmin = substr($strAdmin, 0, 16);
+        $permitted_chars = '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ';
```
Comment (framgiaciv3) [inline app/Repositories/Link/LinkRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/Link/LinkRepository.php</br>**Line**: 28</br>Variable "permitted_chars" is not in valid camel caps format

File: app/Repositories/Link/LinkRepository.php [hunk 4]
```diff
@@ -17,16 +17,16 @@ public function __construct(Link $link)
...
+        $tokenAdmin = substr(str_shuffle($permitted_chars), 0, 16);
```
Comment (framgiaciv3) [inline app/Repositories/Link/LinkRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/Link/LinkRepository.php</br>**Line**: 29</br>Variable "permitted_chars" is not in valid camel caps format

File: app/Repositories/Link/LinkRepository.php [hunk 5]
```diff
@@ -17,16 +17,16 @@ public function __construct(Link $link)
 
     public function createLinkUser()
     {
-        $strUser = md5(time().'user');
-        $tokenUser = substr($strUser, 0, 16);
+        $strDefault = '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ';
```
Comment (tranducquoc) [inline app/Repositories/Link/LinkRepository.php]: chuỗi default này setting đi em

**Pull request merged**